src/lib/rest/httpRequestSend.cpp

```diff
@@ -168,6 +168,7 @@ int httpRequestSendWithCurl
    const std::string&     orig_content_type,
    const std::string&     content,
    const std::string&     fiwareCorrelation,
+   const std::string&     notifyFormat,
```

Comment:
If that is the interpretation, then probably a better name is `ngisv2AttrFormat` or something closer to the hadear string nature. When somebody reads `notifyFormat` tends to thing in the `NotificationFormat` enum and it may be confusing to have a std::string there (that confussion motivates my first comment).